style.css

```diff
@@ -398,3 +444,172 @@ textarea:focus {
 .contact-me-form-button {
   align-self: flex-start;
 }
+
+/* Desktop version of site */
+@media screen and (min-width: 768px) {
+  .logo > a {
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #6070ff;
+    margin: 0 0 0 100px;
+    font-size: 18px;
+  }
+
+  .headline-main {
+    border-radius: 0 0 0 4em;
+    background-color: #fff;
+    background-image: url(./assets/Header\ bg.png);
+    background-size: 96%;
+    background-position-x: center;
+    background-repeat: no-repeat;
+    height: 100vh;
+  }
+
+  .headline {
+    margin: 10px 0;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 12px;
+    width: 751px;
+    height: 268px;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .nav-links {
+    position: absolute;
+    top: 8px;
+    right: 120px;
+    display: flex;
+    margin-top: 15px;
+    padding: 0;
+    color: #6070ff;
+    font-size: 18px;
+  }
+
+  .nav-link {
+    margin: 0 10px;
+    cursor: pointer;
+    color: #344563;
+    font-size: 15px;
+  }
+
+  .social-icons {
+    margin-block: 10px;
+  }
+
+  header {
+    padding: 23px;
+    padding-bottom: 75px;
+  }
+
+  .headline-heading {
+    align-items: center;
+    color: #172b4d;
+  }
+
+  .works-card > img {
+    width: 100%;
+    height: 400px;
+  }
+
+  .works-card:nth-child(even) > img {
+    order: 2;
+  }
+
+  .works-card:nth-child(even) > .works-card-main {
+    order: 1;
+  }
+
+  .works-card {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    column-gap: 2.5em;
+    padding: 1em;
+  }
```

Comment:
- Kindly, add spaces between the project description, tech list and see project button to stick the given design and have a better UI 👍 
### Your Design 👇
<img width="863" alt="Screenshot 2023-06-14 200314" src="https://github.com/Afsheen96/Portfolio/assets/52010789/7281cf68-d5c5-443c-a460-23ce30a7500a">

### Figma Design 👇
<img width="782" alt="Screenshot 2023-06-14 202339" src="https://github.com/Afsheen96/Portfolio/assets/52010789/fa0b65fc-2f3e-48e6-8863-65e93b196b9a">